### PR TITLE
Fix formatting

### PR DESCRIPTION
**Pull request by sypets**

### CODE REVIEW

**Review by linawolf — COMMENTED** [review]
We are trying to get rid of the double double Collin syntax. Please always use explicit code-blocks

**Comment by sypets** [comment]
ok wasn't aware of that. Will fix. There are several other places with double colon but would rather do that separately.

File: Documentation/Appendix/SettingUpTypo3Ddev.rst [hunk 1]
```diff
@@ -74,11 +74,16 @@ Edit configuration
 
 Edit the configuration file :file:`.ddev/config.yaml`.
 
-Set correct PHP version, for example::
+Set correct PHP version, for example:
+
+.. code-block:: yaml
```
Comment (linawolf) [inline Documentation/Appendix/SettingUpTypo3Ddev.rst:79]:
```suggestion
.. code-block:: yaml
   :caption: .ddev/config.yaml
```

File: Documentation/Appendix/SettingUpTypo3Ddev.rst [hunk 2]
```diff
@@ -74,11 +74,16 @@ Edit configuration
...
 
    php_version: "8.1"
 
-Add necessary packages for the :ref:`yarn build process <Yarn build process>` (only needed if you are working on assets):
+Add necessary packages for the :ref:`yarn build process <Yarn build process>`
+(only needed if you are working on assets without using runTests.sh):
+
+.. code-block:: yaml
```
Comment (linawolf) [inline Documentation/Appendix/SettingUpTypo3Ddev.rst:86]:
```suggestion
.. code-block:: yaml
   :caption: .ddev/config.yaml
```

**Comment by sypets** [comment]
We can close here. I integrated the change in my other patch.

**Pull request closed without merging**